setup.py

```diff
@@ -52,11 +52,13 @@ def find_version(*file_paths):
         'Natural Language :: English',
         'License :: OSI Approved :: Apache Software License',
         'Programming Language :: Python',
+        "Programming Language :: Python :: 3 :: Only",
```

Comment:
1. nit - We should use single quotes to keep it consistent with the other classifiers.
2. Should we remove the existing `'Programming Language :: Python :: 3'`? What is the reason for keeping it in addition to the new `"Programming Language :: Python :: 3 :: Only"`?

Reply:
Yep, fair enough, good catch. This got copy pasted and we don't normalize quotes with black.

Reply:
[7518e68](https://github.com/boto/botocore/pull/2967/commits/7518e68f073c86c3dc3d5ee48bbae4e3f4ee6d17)